scripts/cord19/citation_graph.py

```diff
@@ -0,0 +1,45 @@
+import pyserini.collection
+import json
+import csv
+collection = pyserini.collection.Collection(
+    'Cord19AbstractCollection', 'collections/cord19')
+articles = collection.__next__()
+article = None
+with open("node.csv", 'w') as file1:
+    writer = csv.writer(file1)
+    writer.writerow(['Id'])
+with open("edge.csv", 'w') as file2:
+    writer = csv.writer(file2)
+    writer.writerow(['Source', 'Target'])
+
+for (i, d) in enumerate(articles):
+    article = pyserini.collection.Cord19Article(d.raw)
+    # if i == 100:     #change this number
```

Comment:
what does this mean?

Reply:
In case we don't want to run the entire dataset. I delete this in new commit.